src/main/webapp/app/components/category-selector/category-selector.module.ts

```diff
@@ -4,11 +4,10 @@ import { ArtemisSharedModule } from 'app/shared';
 import { ArtemisColorSelectorModule } from 'app/components/color-selector/color-selector.module';
 import { TagInputModule } from 'ngx-chips';
 import { ReactiveFormsModule } from '@angular/forms';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { CategorySelectorComponent } from 'app/components/category-selector/category-selector.component';
 
 @NgModule({
-    imports: [ArtemisSharedModule, ArtemisColorSelectorModule, ReactiveFormsModule, TagInputModule, BrowserAnimationsModule],
+    imports: [ArtemisSharedModule, ArtemisColorSelectorModule, ReactiveFormsModule, TagInputModule],
```

Comment:
The BrowserModule and BrowserAnimationModule should only be imported once in the app module (https://angular.io/guide/ngmodule-faq#should-i-import-browsermodule-or-commonmodule).